packages/cashier/src/Sass/payment-agent-list.scss

```diff
@@ -89,13 +101,6 @@
                 color: var(--text-less-prominent);
             }
         }
-        &-line {
```

Comment:
@hamid-deriv Also there is one other style regarding the removed div for mobile, could you omit that one as well.